README.md

```diff
@@ -1 +1,42 @@
-# whosonfirst
+# Who's on First Data Importer
+
+whosonfirst is a tool used for importing [Who's On First data](https://whosonfirst.mapzen.com/) from local files into a Pelias ElasticSearch repository.
```

Comment:
database might be a better word than repository. repository makes me think of version control things